launchable/test_runners/cucumber.py

```diff
@@ -216,36 +216,37 @@ def parse_func(self, report_file: str) -> Generator[CaseEventType, None, None]:
             click.echo("Can't find test reports from {}. Make sure to confirm report file.".format(
                 report_file), err=True)
 
-        background_test_case_info = None
         for d in data:
             file_name = d.get("uri", "")
             class_name = d.get("name", "")
+            # cucumber can define multiple Background steps
```

Comment:
Are you sure? I don't think we can have multiple Background.

> You can only have one set of Background steps per Feature or Rule. If you need different Background steps for different scenarios, consider breaking up your set of scenarios into more Rules or more Features.

https://cucumber.io/docs/gherkin/reference/#background

Reply:
Ah, sorry, I have to write “multiple steps of a Background “

Reply:
Sorry, I still don't understand what you said. In which case do we need to unify Background durations?

Reply:
`unified_backgrounds` variable is assigned `None` at https://github.com/launchableinc/cli/pull/720/files#diff-959e6ab7da948bb1bcd21d094e6921c6b5d9027f75db90e13901c16753f689d5R243.

Reply:
Thus, I believe that there is no case that we need to unify Background durations. Am I missing something?

Reply:
I understood what you wanted to say. I'll update it, please wait my fixing